roles/matrix-bridge-mautrix-whatsapp/templates/config.yaml.j2

```diff
@@ -105,19 +95,16 @@ bridge:
     default_bridge_receipts: true
     default_bridge_presence: true
     # Servers to always allow double puppeting from
-    double_puppet_server_map: "{{ matrix_domain }}"
-
+    double_puppet_server_map:
+        "{{ matrix_mautrix_whatsapp_homeserver_domain }}": {{ matrix_mautrix_whatsapp_homeserver_address }}
     # Allow using double puppeting from any server with a valid client .well-known file.
     double_puppet_allow_discovery: false
     # Shared secrets for https://github.com/devture/matrix-synapse-shared-secret-auth
     #
     # If set, double puppeting will be enabled automatically for local users
     # instead of users having to find an access token and run `login-matrix`
     # manually.
-    login_shared_secret_map:
-        "{{ matrix_mautrix_whatsapp_homeserver_domain }}": {{ matrix_mautrix_whatsapp_login_shared_secret|to_json }}
-
-
+    login_shared_secret_map: {{ matrix_mautrix_whatsapp_login_shared_secret_map|to_json }}
```

Comment:
Given that this is in the `bridge` section of the configuration, it should probably be named `matrix_mautrix_whatsapp_bridge_login_shared_secret_map` (not `matrix_mautrix_whatsapp_login_shared_secret_map`).

Reply:
Okay, schould i rename `matrix_mautrix_whatsapp_login_shared_secret:  ' '` also?

Reply:
I think `matrix_mautrix_whatsapp_login_shared_secret` is OK and it's an existing variable that people are already using. I suppose we assign to it from `group_vars/matrix_servers` too.